jenkins/helper/site_config.py

```diff
@@ -71,7 +71,14 @@ def get_workspace():
     TEMP = TEMP / 'tmp'
 else:
     TEMP = TEMP / 'ArangoDB'
-if not TEMP.exists():
+if TEMP.exists():
+    # pylint: disable=broad-except
+    try:
+        shutil.rmtree(TEMP)
+        TEMP.mkdir(parents=True)
+    except Exception as ex:
+        print(f"failed to clean temporary directory: {ex} - will continue anyways")
+else:
```

Comment:
Why is it safe to continue? Sounds to me like erroring out here is the right thing to do.